.gitattributes

```diff
@@ -0,0 +1,10 @@
+/tests export-ignore
+/docker export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/docker-compose.yaml export-ignore
+/infection.json export-ignore
+/Makefile export-ignore
+/phpunit.xml.dist export-ignore
+/psalm.xml export-ignore
```

Comment:
please sort these entries by name, ie. move `tests` and `docker` to their places